cluster-operator/src/main/java/io/strimzi/operator/cluster/model/VolumeUtils.java

```diff
@@ -145,19 +151,20 @@ public static Volume createSecretVolume(String name, String secretName, Map<Stri
                 .withName(validName)
                 .withSecret(secretVolumeSource)
                 .build();
-        log.trace("Created secret Volume named '{}' with source secret '{}'", validName, secretName);
+        RECONCILIATION_LOGGER.trace(reconciliation, "Created secret Volume named '{}' with source secret '{}'", validName, secretName);
```

Comment:
Are log messages like this really worth making the code much harder to read and passing reconciliation through 10 different methods?

Reply:
I agree, these are too low value to be worth the noise.